aspnetBlazor/.vscode/tasks.json

```diff
@@ -2,7 +2,8 @@
     "version": "2.0.0",
     "tasks": [
         {
-            "label": "build-torizon-local",
+            "label": "build-application",
```

Comment:
In this tasks would be nice to have `local` in the name also. Like `build-local-application` or `build-application-local`